File: Python/chatyusha.py
# --------------------
# * LICENSE : MIT
# * Author : Chatyusha
# --------------------

import random

DDSK = ["ドド","スコ"]
ddsk_tmp = "ドドスコスコスコ" * 3

judge = ""

flag = False

n=0
while(flag == False):
    t = random.randint(0,1) % 2
    print(DDSK[t])
    judge += DDSK[t]

    if len(judge) >= 24:
        if judge[2*n:] == ddsk_tmp:
            print("ラブ注入♡")
            flag = True
        n += 1

File: C#/Chatyusha.cs
// --------------------
// * LICENSE : MIT
// * Author : Chatyusha
// --------------------
using System;
using System.Collections;
using System.Collections.Generic;

namespace DDSK
{
    class Program
    {
        
        static void Main()
        {
            bool Flag = false;

            int counter = 0;

            string DDSKTmp = "ドドスコスコスコドドスコスコスコドドスコスコスコ";

            List<string> DDSKList = new List<string>();

            string DDSKString = "";
            DDSKList.Add("ドド");
            DDSKList.Add("スコ");

            while(Flag == false)
            {
                var gen = new Random();
                var unit =DDSKList[gen.Next(0,2)%2];
                DDSKString += unit;
                Console.WriteLine(unit);
                if(DDSKString.Length >= 24)
                {
                    if(DDSKString.Substring(counter*2,24) == DDSKTmp)
                    {
                        Flag = true;
                        Console.WriteLine("ラブ注入♡");
                    }
                    counter++;
                }
            }
        }
    }
}